.github/chainguard/neko.sts.yaml

```diff
@@ -0,0 +1,12 @@
+issuer: https://accounts.google.com
+# neko77@neko77.iam.gserviceaccount.com
+subject: "113809027484440661352"
+permissions:
+  discussions: read
+  issues: read
+  pull-requests: read
```

Comment:
this should be 
```suggestion
  pull_requests: read
```

Reply:
Thanks for the review @cpanato !

Are these the [permissions to reference](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#:~:text=You%20can%20define%20the%20access%20that%20the%20GITHUB_TOKEN%20will%20permit%20by%20specifying%20read%2C%20write%2C%20or%20none%20as%20the%20value%20of%20the%20available%20scopes%20within%20the%20permissions%20key)?   I'm happy to make the change but I'm wondering if you can point me at the authoritative source because I have a feeling I'll interact with this service a lot.  :)

Reply:
It is the JSON encoding here: https://pkg.go.dev/github.com/google/go-github/v59/github#InstallationPermissions